ipinfo/cmd_tool.go

```diff
@@ -38,6 +39,7 @@ Commands:
   prev         get the previous IP of the input IP
   is_v4        reports whether input is an IPv4 address.
   is_v6        reports whether input is an IPv6 address.
+  isValid      reports whether the Addr is an initialized address (not the zero Addr).
```

Comment:
We can't be using internal technical jargon in the user-facing help message

```suggestion
  isValid      reports whether an IP is valid.
```